### DISCUSSION TITLE
Freeze the first 5 columns in each of the 3 grids and need to do horizontal scroll simultaneously in p tables

### DISCUSSION THREAD

**Discussion by tusharswain2022** [primefaces/community#383] [PrimeNG] [answered]
Hi, 
I have three individuals p-tables and they have three horizontal scroll bars within the p-tables they  have frozen columns so i need to achieve 
Simultaneously horizontal scrolling for example if I move horizontal scroll bar of one table other two tables horizontal scroll also move without
breaking frozen columns how  I can achieve within angular ng prime?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by tusharswain2022** [comment]
Hi @cetincakiroglu 

As per your comment  This  is stackblitz  example

https://stackblitz.com/edit/koi9ab?file=src%2Fapp%2Fdemo%2Ftable-frozen-columns-demo.html

how can i achieve  simultaneously  horizontal scroll bar if i move table 1 horizontal scroll it moves table2 horizontal scroll bar vice versa

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, table doesn't have such a feature. I'm guessing but maybe this can help you:

1-) Access the table elements by viewChild
2-) Listen the scrolling of table 1 and get the scrolling data.
3-) invoke `scrollTo` by providing scrolling data.

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
